src/PrestaShopBundle/Controller/Admin/Sell/Catalog/DiscountController.php

```diff
@@ -232,6 +234,42 @@ public function deleteAction(int $discountId): RedirectResponse
         return $this->redirectToRoute('admin_discounts_index');
     }
 
+    /**
+     * Get all customer groups formatted for the component.
+     * Since customer groups don't have a parent group concept, we create a single "group"
```

Comment:
I don't like this at all. We are doing a hackjob by faking some data to use a component intended for something else